### DISCUSSION TITLE
Tortoise Voices not Recognized

### DISCUSSION THREAD

**Discussion by quantumlump** [rsxdalv/TTS-WebUI#408] [Q&A] [answered]
I made sure all of the correct voices folders with the correct names were in the voices folder but the terminal is saying it cant find the voice. Is there something I am doing wrong? Is there something I can do to make the voices and models download automatically like Bark?
![sdf](https://github.com/user-attachments/assets/b60995b3-43c9-471a-aa08-eaf458e49b20)

**Comment by rsxdalv** [comment]
I'm fairly certain you are doing it correctly but gradio's new 5th version is just once again breaking stuff.

**Reply by rsxdalv** [reply, marked as answer]
Are you using the react ui? If so, could you try to see if the same error happens in the gradio UI?

**Comment by quantumlump** [comment]
Thanks for the quick response! It does in fact work in Gradio, thanks for the help

**Comment by rsxdalv** [comment]
I created an issue for this bug: https://github.com/rsxdalv/tts-generation-webui/issues/417

I will try to fix the React UI in due time.
